webui/ocw/lib/vault.py

```diff
@@ -21,6 +26,7 @@ def __init__(self, vault_namespace):
         self.namespace = vault_namespace
         self.password = cfg.get(['vault', 'password'])
         self.certificate_dir = cfg.get(['vault', 'cert_dir'], '/etc/ssl/certs')
+        self.use_file_cache = cfg.getBoolean(['vault', 'use_file_cache'])
```

Comment:
so basically to change this you need to restart server , right ? 
what if we would just check config file every time so basically you can change this setting on the fly ?

Reply:
We currently read config file on the fly see https://github.com/cfconrad/pcw/blob/master/webui/webui/settings.py#L184

The problem is, it will only be used if you get "new" credentials and for this you would need to wait that 2h. 
But when you change the code during development, django already reload the server. So you get once new credentials and then these are stored into the cache file.

Reply:
> The problem is, it will only be used if you get "new" credentials and for this you would need to > wait that 2h.
that was wrong, as you correctly mentioned, the constructor where we store it in `self.use_file_cache` is only called once. So yes I will change it.